inst/templates/ad_advs.R

```diff
@@ -180,7 +181,7 @@ who_wt_for_lgth <- who_wt_for_lgth_boys %>%
 # as needed and assign to the variables below.
 # For illustration purposes read in admiral test data
 
-data("vs_peds")
+vs_peds <- vs_peds
```

Comment:
Is this line of code necessary ?

Reply:
you're probably right that this could be removed

Reply:
@zdz2101 don't we still need the `data("xxx")` lines though to load the data from the pharmaversesdtm package? see https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_advs.R for example (lines 18 and 19)